pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[tool.poetry]
+name = "icalevents"
+version = "0.1.26"
+description = "Simple Python 3 library to download, parse and query iCal sources."
+authors = ["Martin Eigenmann <github@eigenmannmartin.ch>", "Thomas Irgang <thomas@irgang.eu>", "Alexander Hultnér"]
```

Comment:
You can use ahultner+github@gmail.com for my email

Reply:
did not find any email on your profile and homepage - done